### PR TITLE
Test aliases conlict handling

### PR DESCRIPTION
**Pull request by pkratoch**
Requires: https://github.com/rpm-software-management/ci-dnf-stack/pull/691 - specifically, commit "Define new tag ``@no_installroot``"

### CODE REVIEW

**Comment by pkratoch** [comment]
Also requires https://github.com/rpm-software-management/ci-dnf-stack/pull/715

File: dnf-behave-tests/features/alias-command.feature [hunk 1]
```diff
@@ -32,7 +36,57 @@ Scenario: Delete alias
    When I execute dnf with args "alias list"
    Then the exit code is 0
    And stdout does not contain "Alias inthrone"
```
Comment (jrohel) [inline dnf-behave-tests/features/alias-command.feature]: Or stdout is
"""
No aliases defined.
"""

Comment (pkratoch) [reply dnf-behave-tests/features/alias-command.feature]: I added commit that changes all stdout checks in this feature file.

**Comment by jrohel** [comment]
LGTM

**Pull request merged**